Source/Evolve/WarpXEvolve.cpp

```diff
@@ -305,6 +305,8 @@ WarpX::OneStep_nosub (Real cur_time)
     doQEDEvents();
 #endif
 
+    mypc->doResampling(istep[0]+1);
```

Comment:
Why do we need to pass `istep[0]+1` instead of `istep[0]` ? I would suggest to add a short comment to explain why this is necessary...